internal/cmd/start.go

```diff
@@ -76,15 +79,17 @@ func startCmd() *cobra.Command {
 		Use:   "start",
 		Short: "start the spinup API and frontend servers",
 		Run: func(cmd *cobra.Command, args []string) {
-			log.Println(fmt.Sprintf("INFO: using config file: %s", cfgFile))
+			utils.InitializeLogger("", "")
```

Comment:
do we need this here? Looks like we initializing on line https://github.com/spinup-host/spinup/pull/115/files#diff-032043c7b9694777926f5fa9af41a34f3df0230518bf749b08df4f2a47fecd32R88